gfx/drivers_context/sdl_gl_ctx.c

```diff
@@ -87,7 +87,9 @@ static void sdl_ctx_destroy(void *data)
       return;
 
    sdl_ctx_destroy_resources(sdl);
+#ifndef WEBOS
    if (sdl->subsystem_inited)
```

Comment:
I think it would make sense to have some comment here (and possibly even some further investigation at some point)

I am not entirely sure how SDL init/reinit/deinit flow works (maybe some RetroArch SDL expert may give some thoughts) but in our case, without an explicitly shutdown of SDL Video subsystem EGL context can't be recreated later when starting up a core.